packages/web/src/components/search/DataSearch.js

```diff
@@ -1065,8 +1079,13 @@ class DataSearch extends Component {
 															highlightedIndex,
 															index,
 														),
+														justifyContent: 'flex-start',
 													}}
 												>
+													<div style={{ padding: '0 10px 0 0' }}>
+														{sugg.source._recent_search && <RecentSvg className={getClassName(this.props.innerClass, 'recent-search-icon') || null} icon={recentSearchesIcon} />}
```

Comment:
@anik-ghosh-au7 What if the suggestion doesn't have `source` property. It might be possible when the user defines their own default suggestions. Please do proper type checks.